concepts/peg-stability-mechanisms.md

```diff
@@ -1,5 +1,35 @@
-# TBC
+# Peg Stability Mechanisms
 
-{% hint style="info" %}
-Sorry! We haven't been able to publish documentation on this topic yet. Please check back soon.
-{% endhint %}
+## Introduction
+
+The Endgame Plan allows the Maker Protocol to dynamically impact the supply and demand of DAI. These mechanisms are called Peg Stability Mechanisms. Note that should not be confused with the similarly-named [Peg Stability Modules (PSMs)](https://manual.makerdao.com/module-index/module-psm)
```

Comment:
Should change this imo. It implies that these are due to the endgame plan, which is not really true.

Reply:
Run grammarly on something, second sentence parses strangely.